src/pages/case-studies.pug

```diff
@@ -30,8 +30,9 @@ block pageComponents
           }
         },
         presentationImage: {
-          src: "/assets/images/case-studies/laced/presentation.png",
-          alt: "Laced"
+          src: "/assets/images/case-studies/hx/presentation.webp",
```

Comment:
We don't need to include `webp` images ourselves: our setup will generate them for us. As a result, no need for the fallback (just use the PNG).